src/main/io/rc_controls.c

```diff
@@ -342,7 +343,16 @@ void updateActivatedModes(modeActivationCondition_t *modeActivationConditions)
         modeActivationCondition_t *modeActivationCondition = &modeActivationConditions[index];
 
         if (isRangeActive(modeActivationCondition->auxChannelIndex, &modeActivationCondition->range)) {
```

Comment:
yeah, i was also thinking about this problem and had already decided it needs a bit more work.